docs/tutorials/06_qubit_mappers.ipynb

```diff
@@ -0,0 +1,476 @@
+   "outputs": [],
+   "source": [
+    "converter = QubitConverter(ParityMapper(), two_qubit_reduction=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6bdce1e0",
+   "metadata": {},
+   "source": [
+    "In order for the reduction to work, you now also need to supply the number of particles:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "569a3b78",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "-1.0523732457728605 * II\n",
+      "+ 0.39793742484317896 * IZ\n",
+      "- 0.39793742484317896 * ZI\n",
+      "- 0.01128010425623538 * ZZ\n",
+      "+ 0.18093119978423122 * XX\n"
+     ]
+    }
+   ],
+   "source": [
+    "qubit_op = converter.convert(fermionic_op, num_particles=problem.num_particles)\n",
+    "print(qubit_op)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2b4b1bec",
+   "metadata": {},
+   "source": [
+    "### More advanced qubit reductions\n",
+    "\n",
+    "The `QubitConverter` also enables you to perform more advanced qubit reductions, which are based on finding Z2 symmetries in the Hilbert space of the qubit. A requirement for this to be useful, is that you know in which symmetry-subspace you need to look for your actual solution of interest. This can be a bit tricky, but luckily the problem classes of Qiskit Nature provide you with a utility to automatically determine that correct subspace.\n",
+    "\n",
+    "Here is how you can use this to your advantage:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "4270e7cf",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "-1.0410931415166251 * I\n",
+      "- 0.7958748496863578 * Z\n",
+      "+ 0.1809311997842312 * X\n"
+     ]
+    }
+   ],
+   "source": [
+    "converter = QubitConverter(JordanWignerMapper(), z2symmetry_reduction=\"auto\")\n",
+    "\n",
+    "qubit_op = converter.convert(fermionic_op, sector_locator=problem.symmetry_sector_locator)\n",
+    "print(qubit_op)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "93639c5b",
+   "metadata": {},
+   "source": [
+    "As you can see here, the H2 molecule is such a simple system that we can simulate it entirely on a single qubit!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "48ff72a5",
+   "metadata": {},
+   "source": [
+    "## The `QubitConverter`\n",
+    "\n",
+    "We would like to end this tutorial with a few more notes on the `QubitConverter`.\n",
+    "\n",
+    "There are multiple reasons for its existence:\n",
+    "- it handles the qubit reduction techniques which are applied on the qubit space *after* the mapping process\n",
+    "- it **stores** the symmetry information which you find during the conversion of your *Hamiltonian* such that you can recall and use the same symmetry reductions when mapping additional observables later one\n",
```

Comment:
... when mapping additional observables later one

At the end should be `on` not `one`

I imagine this can be expanded with the other mappers. Though I guess we might want to think about this in regards of document structure going forwards.

Reply:
Yeh that's a good point. Maybe this tutorial could become more of a general guide on how to use our mappers whereas for each specific one there is its own documentation page + maybe a HowTo for discussing some more examples? :thinking:

But this can be done later :+1: